test/services/idm/fetch-user-roles-and-groups.service.test.js

```diff
@@ -26,9 +26,11 @@ describe('Fetch User Roles And Groups service', () => {
   let testGroup
 
   beforeEach(async () => {
-    await DatabaseSupport.clean()
-
-    testUser = await UserHelper.add()
+    testUser = await UserHelper.add(
+      {
+        username: `${generateUUID()}@test.com`
```

Comment:
Was tempted to use faker but this does the trick. We just need a unique name but don't want to replicate the bfe multiple times. Could look to nest / create a simple random text func